libs/api/Verifier.cpp

```diff
@@ -69,6 +95,74 @@ _get_program_and_map_names(
     }
 
     ELFIO::const_symbol_section_accessor symbols{reader, reader.sections[".symtab"]};
+
+    if (reader.sections[".maps"]) {
```

Comment:
This function is getting long so harder to review/maintain over time.  Suggest moving the ".maps" block into a function called from here.

Reply:
Unresolving since I was suggesting moving lines 134-182 into a separate function and calling it from here.
If there's a reason not to do so, let me know.

Reply:
Thanks. It should be easier to follow now.